### ISSUE TITLE
✨ NEW: explain computer PK better

### ISSUE THREAD

**Issue by ltalirz** [aiidateam/aiida-tutorials#202]
Question arose while working on
https://aiida-tutorials.readthedocs.io/en/latest/pages/2019_ISSP_Chiba_Japan/sections/calculations.html#computer-setup

Indeed - `verdi code list` shows PKs, while `verdi computer list` doesn't.
The problem is, of course, that computers are stored in a different table.

I do think we need to find a more prominent place for information on the entities available in aiida than this
https://aiida-core.readthedocs.io/en/latest/concepts/provenance.html#other-entities
In particular, since `Code` are now stored in the `Node` table, but still have a number of dedicated commands (`verdi code`, `load_code`, ...).

mentioning @sphuber for info

**Comment by ltalirz** [comment]
Maybe also worth mentioning @CasperWA since from the export you've become an expert on all the different entities ;-)
An "entity overview" would be very useful I believe

**Comment by mbercx** [comment]
Good point! I'm not sure if we have anything like this in the current material. Maybe something to add to the "Data" section? What do you think, @CasperWA, expert of all thing entity? 😁

**Comment by sphuber** [comment]
The title of this issue is and the statement "The problem is, of course, that computers are stored in a different table." are incorrect: `Computer` instances _do_ have a pk. Any entity in AiiDA has a pk, not just nodes. The discussion in this issue doesn't really seem to address this question though. I suspect the original confusion by the user was why `verdi computer list` does not show the pk. So I doubt that an entity overview would have addressed the original question, still might be useful though. Probably we should think about maybe adding a flag to `verdi computer list` that will show the pk and UUID in addition to the label that is being shown by default

**Comment by ltalirz** [comment]
While adding a `PK` to `verdi computer list` solves part of the issue, another aspect of my point here was that a computer and a node can be assigned the *same* PK, i.e. there are "computer PKs" and "node PKs".

Without an overview of different entities in AiiDA, a new user has no chance of knowing about this structure (e.g., there is no intuitive reason that would allow you to deduce that Code PKs are the same as Node PKs but Computer PKs are different).
